helpers/erc4626.ts

```diff
@@ -0,0 +1,52 @@
+import { ChainApi } from "@defillama/sdk";
+
+// helper for ERC4626-vaults
+// https://docs.openzeppelin.com/contracts/4.x/erc4626
+
+const ERC4626Abis: any = {
+  asset: 'address:asset',
+  decimals: 'uint8:decimals',
+  totalAssets: 'uint256:totalSupply',
+  assetsPerShare: 'function convertToAssets(uint256 shares) view returns (uint256 assets)',
+}
+
+interface ERC4626VaultInfo {
+  asset: string;
+  assetDecimals: number;
+  totalAssets: bigint;
+  assetsPerShare: bigint;
+}
+
+// return info of given ERC4626 vault
+export async function getERC4626VaultInfo(usingApi: ChainApi, vault: string): Promise<ERC4626VaultInfo | null> {
+  const asset = await usingApi.call({
```

Comment:
better to use vault array here, that way we can use `multiCall`and cut down the number of calls